app/components/navigation.tsx

```diff
@@ -30,17 +30,11 @@ export default function Navigation(): JSX.Element {
               Register
             </NavLink>
           </Nav>
-          {/*
-          When implemented authentication
           <Nav className="ms-auto">
             <NavLink to="/profile/:id" className="nav-link">
```

Comment:
The url is actually changed since this was written.
```suggestion
            <NavLink to="/account" className="nav-link">
```